tracer/contrib/gorilla/muxtrace/muxtrace.go

```diff
@@ -106,6 +107,9 @@ func (t *tracedResponseWriter) WriteHeader(status int) {
 	t.w.WriteHeader(status)
 	t.status = status
 	t.span.SetMeta(ext.HTTPCode, strconv.Itoa(status))
+	if status >= 500 && status < 600 {
+		t.span.SetError(errors.New(strconv.Itoa(status)))
+	}
```

Comment:
I think we need to keep `SetError` for real go errors and not artificial ones we create otherwise it's confusing in the UI if we try to format this as a go error which should have a stack trace but has none.
I guess in this case the only thing that is of interest in `SetError` is `t.Span.Error = 1`